rstudio-tidyverse/rockcraft.yaml

```
@@ -0,0 +1,259 @@
+# Based on:
+# https://github.com/kubeflow/kubeflow/blob/v1.10.0-rc.1/components/example-notebook-servers/rstudio-tidyverse/Dockerfile
+# https://github.com/kubeflow/kubeflow/blob/v1.10.0-rc.1/components/example-notebook-servers/rstudio/Dockerfile
+# https://github.com/kubeflow/kubeflow/blob/v1.10.0-rc.1/components/example-notebook-servers/base/Dockerfile
+
+name: rstudio-tidyverse
+summary: An image for using tidyverse
+description: |
+  The tidyverse is an opinionated collection of R packages designed for data science. 
+  Tidyverse are installed in Conda environment, which is automatically activated.
+version: "v1.10.0-rc.1"
+license: Apache-2.0
+base: ubuntu@22.04
+run-user: _daemon_
+
+platforms:
+  amd64:
+
+services:
+  rstudio-tidyverse:
+    override: replace
+    summary: "Rstudio Tidyverse service"
+    startup: enabled
+    command: /usr/lib/rstudio-server/bin/rserver --server-daemonize=0 --server-data-dir=/tmp/rstudio_runtime --server-working-dir=/home/_daemon_ --server-user=_daemon_ --rsession-which-r=/opt/conda/lib/R --www-address=0.0.0.0 --www-port=8888 --auth-none=1 --rsession-path=/opt/rsession.sh --rsession-ld-library-path=/opt/conda/lib
+    working-dir: /home/
+    environment:
+       NB_USER: _daemon_
+       NB_UID: 1000
+       NB_GID: 0
+       NB_PREFIX: /
+       USERS_GID: 100
+       HOME: /home/_daemon_
+       HOME_TMP: /tmp_home/_daemon_
+       USER: _daemon_
+       SHELL: /bin/bash
+       LANG: en_US.UTF-8to
+       LANGUAGE: en_US.UTF-8
+       LC_ALL: en_US.UTF-8
+       PATH: /opt/conda/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+       PYTHONPATH: /opt/conda/lib/python3.11/site-packages/
+       RSTUDIO_RUNTIME_DIR: "/tmp/rstudio_runtime"
+       XDG_RUNTIME_DIR: "/tmp/runtime-_daemon_"
```

suggestion(blocking): I think in this case, it makes sense to follow the upstream Dockerfile and use `jovyan` as the user. We also follow this in other rocks
```suggestion
       HOME: /home/${NB_USER}
       HOME_TMP: /tmp_home/${NB_USER}
       USER: ${NB_UID}
       SHELL: /bin/bash
       LANG: en_US.UTF-8
       LANGUAGE: en_US.UTF-8
       LC_ALL: en_US.UTF-8
       PATH: /opt/conda/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
       PYTHONPATH: /opt/conda/lib/python3.11/site-packages/
       RSTUDIO_RUNTIME_DIR: "/tmp/rstudio_runtime"
       XDG_RUNTIME_DIR: "/tmp/runtime-${NB_USER}"
```

---

Fixed